doc/sphinx/source/notes_7_0.rst

```diff
@@ -21,9 +38,58 @@ CPU drivers
   Please note that these DBKs are still a proof-of-concept and
   are subject to change without notice.
 
+* When built with -DENABLE_CONFORMANCE=ON, PoCL's CPU driver
+  is now able to pass the full OpenCL-CTS conformance testsuite.
+  We have submitted & achieved conformant status on <TODO day>.
+
+* Implemented Windows platform support, using both MinGW and MSVC
+  toolchains.
+
+* Support for generating NSIS package on Windows
+
 * cl_khr_command_buffer implementation has been updated to 0.9.6
   Note that this version is not backwards compatible with previous versions.
 
+* implemented extension cl_ext_buffer_device_address / 1.0.2
+
+* implemented extension cl_khr_command_buffer_mutable_dispatch / 0.9.3
+
+* implemented cl_intel_subgroups_{char/short/long} extensions
+
+* implemented missing intel_sub_groups_shuffle_down/up for scalar types
+
+* implemented cl_intel_subgroup shuffle/xor/up/down with vector arguments
+     (this is required by SYCL-CTS)
+
+* implemented read_imageh
+
+* SPIR-V support has been improved to support additional subgroup-related
+  functions; additionally, SPIR-V up to version 1.5 is now supported
+  with LLVM 20 and up to 1.3 with LLVM 19.
+
+* CPU device's max_mem_alloc_size is now equal to global_mem_size.
+
+* handling of LLVM's unreachable instructions has been improved. The
+  new handling depends on the workitem handler; with "loopvec" the
+  unreachable is optimized away; with CBS the unreachable is converted
+  to set an error flag, which the CPU driver then uses to set the NDRange
+  event's state to failed.
+
+* support for using LLVMSPIRVLib for SPIR-V input translation,
+  instead of llvm-spirv binary
+
+* KERNELLIB_HOST_CPU_VARIANTS now includes "generic" CPU as a fallback.
+
+* printf() now correctly supports all OpenCL vector types.
+  All of OpenCL-CTS printf tests now pass.
+
+* added option to disable support of subnormal floats
+
+* added option to avoid division-by-zero exceptions using a LLVM pass
+  instead of SIGFPE handler (CMake option ENABLE_SIGFPE_HANDLER).
+  This has the advantage of being a platform-independent and
+  scheduler-independent (TBB,OpenMP) solution.
+
```

Comment:
Rematerialization was added.

Reply:
added a short note on remat, you can expand / review